services/rfq/relayer/relconfig/getters.go

```diff
@@ -691,3 +692,17 @@ func (c Config) GetQuoteSubmissionTimeout() time.Duration {
 	}
 	return timeout
 }
+
+const defaultBlockWindow = 5
+
+const defaultVolumeLimit = 10_000
+
+// GetVolumeLimit returns the volume limit for the relayer.
+func (c Config) GetVolumeLimit() float64 {
```

Comment:
Is VolumeLimit meant to be in human-readable units (with decimals) or no? It seems like it's used as non-human-readable here, but in that case we should use `big.Int` instead of `float64`

Though I think less error prone to make this a human-readable value with a getter that does the decimal conversion

See how we do it here: https://github.com/synapsecns/sanguine/blob/master/services/rfq/relayer/relconfig/getters.go#L423